package.json

```diff
@@ -1,21 +1,39 @@
 {
-  "name": "root",
-  "private": true,
-  "devDependencies": {
-    "@types/jest": "^24.0.18",
-    "lerna": "^3.16.4",
-    "lodash.camelcase": "^4.3.0"
-  },
-  "scripts": {
-    "watch": "lerna run watch --parallel --stream",
-    "build": "lerna run build --stream",
-    "build:debug": "lerna run build:debug --stream",
-    "bootstrap": "lerna bootstrap",
-    "pretest": "lerna run build",
-    "test": "lerna run test --stream",
-    "lint": "lerna run lint --stream",
-    "lint:fix": "lerna run lint:fix --stream",
-    "release": "lerna bootstrap && npm run test && npm run build && lerna publish --force-publish=* --conventional-commits",
-    "prerelease": "lerna bootstrap && npm run test && npm run build && lerna publish --dist-tag prerelease --force-publish=* --preid rc"
-  }
+	"name": "root",
+	"private": true,
+	"devDependencies": {
+		"@types/jest": "^24.0.18",
```

Comment:
I don't know why this changed indentation to 4 spaces 🤔

Reply:
The first bootstrap hoist execution removed the 'scripts' section from the file, and I had to manually recover that, so that's probably my fault.